database/index.js

```diff
@@ -1,33 +1,89 @@
-var mysql = require('mysql');
+const mysql = require('mysql');
 
-var connection = mysql.createConnection({
-  host: 'localhost',
+const DBconnection = mysql.createConnection({
   database: 'yumpSF',
-  user: 'root'
+  user: 'root',
 });
 
-var addRestaurant = function (name, callback) {
-  connection.query('INSERT INTO restaurants (name) VALUES (?)', [name], callback);
+const getRestaurantDishes = (restaurantID, callback) => {
+  const queryStr = `SELECT * FROM restaurants INNER JOIN dishes ON (id = restaurant_id) WHERE id = ${restaurantID}`;
```

Comment:
Have you tested this query? I don't remember my sql syntax super well, but it looks like maybe you're selecting restaurants, when it looks like the purpose of this function is supposed to be getting from the dishes table.

Reply:
I just checked another time. This query gives me the dishes for the given restaurant. All works as expected. Thanks!